index.json

```
@@ -1,17 +1,20 @@
 {
-  "aws_access_key_id": "AKIA6KJQR5EGTCVYCLKT",
-  "aws_secret_access_key": "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",
+  "aws_access_key_id": "AKIA6KJQR5EGTCVY",
```

Biome (2.1.2)

[error] The key aws_access_key_id was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)